src/cupbearer/tasks/adversarial_examples.py

```diff
@@ -17,26 +16,30 @@ class AdversarialExampleTask(TaskConfig):
     steps: int = 40
     eps: float = 8 / 255
 
-    def _init_train_data(self):
-        self._train_data = TrainDataFromRun(path=self.path)
+    def _get_clean_data(self, train: bool) -> DatasetConfig:
+        if train:
+            return TrainDataFromRun(path=self.path)
+        else:
+            return TrainDataFromRun(path=self.path).get_test_split()
 
-    def _get_anomalous_test_data(self):
+    def _get_anomalous_data(self, train: bool) -> DatasetConfig:
```

Comment:
```suggestion
    def _get_anomalous_data(self) -> DatasetConfig:
        return self._get_anomalous_data_by_split(train=True)

    def _get_anomalous_test_data(self) -> DatasetConfig:
        return self._get_anomalous_data_by_split(train=False)

    def _get_anomalous_data_by_split(self, train: bool) -> DatasetConfig:
```
Something like this to match signature in parent class.